tests/foreman/api/test_rhci.py

```diff
@@ -0,0 +1,123 @@
+"""Unit tests for the ``activation_keys`` paths."""
+from ddt import data, ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=R0904
+
+
+@ddt
+class RHCIDeploymentTestCase(APITestCase):
+    """Tests for the RHCI deployment API."""
+
+    @data(
+        gen_string(str_type='alpha'),
+        gen_string(str_type='alphanumeric'),
+        gen_string(str_type='cjk'),
+        gen_string(str_type='latin1'),
+    )
+    def test_positive_create_deployment(self, data):
+        """@Test: Create a simple deployment.
+
+        @Assert: An RHCI deployment is created with a random name.
+
+        @Feature: RHCI Deployment
+
+        """
+        try:
+            attrs = entities.RHCIDeployment(
+                name=data,
+                organization_id=1,
+                lifecycle_environment_id=2,
+                deploy_rhev=True,
+                rhev_engine_host_id=2,
+                rhev_storage_type='NFS',
+                rhev_engine_admin_password='changeme',
+            ).create_json()
+        except HTTPError as err:
+            self.fail(err)
+        self.assertIsInstance(attrs['deployment']['id'], int)
```

Comment:
hmm... I could assert the name is the same. I originally opted for the id since it was the one value that i didn't explicitly pass in. But in light of that bug you pointed me to, a name assertion would be more reliable.